### DISCUSSION TITLE
What does this mean

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [KadeArchive/Kade-Engine#2197] [General]
source/PlayState.hx:1108: characters 2-8 : Expected }

For reference here is the code

1107		#if cpp
1108		public static var luaModchart:ModchartState = null;
1109		#end

**Comment by NK-523** [comment]
It means a line 1108 there is a expected, but missing }. I think.

**Reply by THEkatinamicrowave** [reply]
but where do I put that }

**Reply by NK-523** [reply]
a snip from my playstate.hx,
![image](https://user-images.githubusercontent.com/77185327/132719263-01b99c54-960f-41bc-ba7b-5e6300aa36b9.png)
are you on a different kade engine than 1.7, your code is a few lines up.

**Reply by THEkatinamicrowave** [reply]
I use 1.7 why
Do you think I added something below?
I'll check
Nope nothing came up

**Reply by NK-523** [reply]
I use 1.7 as well, that is from my playstate.hx.

**Reply by NK-523** [reply]
try mimicking my playstate.hx image.

**Reply by THEkatinamicrowave** [reply]
trying
everything is the same
